dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils1_1.java

```diff
@@ -159,8 +164,37 @@ private static String jsonifyGCEvent(int maxFieldLength, String wlpUserDir,
         return sb.toString();
     }
 
+    private static String jsonifyGCEvent(String wlpUserDir, String serverName, String hostName, Object event, String[] tags) {
+        GCData gcData = (GCData) event;
+
+        StringBuilder sb = CollectorJsonHelpers.startGCJson1_1(hostName, wlpUserDir, serverName);
+
+        CollectorJsonHelpers.addToJSON(sb, gcData.getHeapKey1_1(), Long.toString(gcData.getHeap()), false, true, false, false, false);
```

Comment:
shouldn't need to json escape the values of GCData (applies also to same method in CollectorJsonUtils class)